bors.toml

```diff
@@ -1,6 +1,7 @@
 status = [
   'check-links',
-  'check-style'
+  'check-style',
+  'vale'
```

Comment:
I know this is not the point of this PR but you could add the builds in the bors toml as well. 

```
check-builds (14)
check-builds (16)
```

Reply:
Does it have to be 
```
check-builds (14)
check-builds (16)
```

Or will just `check-builds` work too?

Reply:
It has to be as its the names of the jobs
![Uploading Screenshot 2022-08-09 at 13.57.46.png…]()


```
check-builds (14)
check-builds (16)
```